tests/test_costs.py

```diff
@@ -1,9 +1,9 @@
-import pytest
 import numpy as np
-from ruptures.costs import cost_factory, CostLinear
+import pytest
+from ruptures.costs import CostLinear, cost_factory
+from ruptures.costs.costnormal import CostNormal
```

Comment:
```suggestion
from ruptures.costs import CostLinear, CostNormal, cost_factory
```

Reply:
good catch, solved in bcfaae30993b9e649b147ad12a3c828c044091c2